.erb-lint.yml

```diff
@@ -0,0 +1,74 @@
+---
+linters:
+  # Ensures that all <% tags have a closing tag.
+  SelfClosingTag:
+    enabled: true
+  # Ensures that all ending tags use the -%> notion.
+  RightTrim:
+    enabled: true
+    enforced_style: '-'
+  # Trailing whitespace in ERB templates
+  TrailingWhitespace:
+    enabled: true
+  # Ensure that
+  # ```
+  # <%
+  #  code
+  # -%>
+  # ```
+  # notion is followed
+  # Note: This will not check for proper code indentation.
+  ClosingErbTagIndent:
+    enabled: true
+  # Ensure there's a newline at the end of the file.
+  FinalNewline:
+    enabled: true
+  # Run Rubocop on code in tags
+  Rubocop:
+    enabled: true
+    rubocop_config:
+      inherit_from:
+        - .rubocop.yml
+      # erb-lint guidelines recommended turning these cops off.
+      # They don't apply to ERB tags.
+      Layout/InitialIndentation:
+        Enabled: false
+      Layout/TrailingBlankLines:
+        Enabled: false
+      Layout/TrailingWhitespace:
+        Enabled: false
+      Naming/FileName:
+        Enabled: false
+      Style/FrozenStringLiteralComment:
+        Enabled: false
+      Metrics/LineLength:
+        Enabled: false
+      Lint/UselessAssignment:
+        Enabled: false
+      Rails/OutputSafety:
+        Enabled: false
+  # ---------------------------------------
+  # Turning off unnecessary default linters
+  # ---------------------------------------
+  # Terraform uses tabs.
```

Comment:
We use tabs in some places and spaces in others. We should make sure we are consistent and turn this flag on. Do you adding a TODO to the issue I have just created about this:
https://github.com/GoogleCloudPlatform/magic-modules/issues/218

Reply:
Added.